utils/remote_control/navigator_joystick_control/nodes/navigator_joystick.py

```diff
@@ -5,53 +5,56 @@
 import numpy as np
 import math
 import tf
-from geometry_msgs.msg import WrenchStamped, Twist, Point
+import itertools
+from navigator_tools import make_header
+from geometry_msgs.msg import WrenchStamped, Twist, PoseStamped
 from sensor_msgs.msg import Joy
 from nav_msgs.msg import Odometry
 from navigator_msg_multiplexer.srv import wrench_arbiter
+from kill_handling.broadcaster import KillBroadcaster
 from sub8_alarm import AlarmBroadcaster
 
-rospy.init_node("joystick", log_level = rospy.DEBUG)
+rospy.init_node("joystick")
 
-class JOYSTICK(object):
+class Joystick(object):
     # Base class for whatever you are writing
     def __init__(self):
 
         self.force_scale = rospy.get_param("~force_scale")
         self.torque_scale = rospy.get_param("~torque_scale")
 
-        self.last_controller_state = 0;
-        self.last_station_hold_state = 0;
-        self.last_kill_state = 0
-        self.last_revive_state = 0
-        self.current_pose = Odometry()
+        self.last_change_mode = 0
```

Comment:
The joystick returns ints as the button states so it's easier to compare and set that way